### DISCUSSION TITLE
Full customization of Arrowskin properties (including >18k)

### DISCUSSION THREAD

**Discussion by Ronezkj15** [Leather128/LeathersFunkinEngine#53] [Ideas]
Since the Arrowskin's properties have already been coded out and filled with non-hardcoded sizing and positioning, the next step should be full customization in the amount of keys possible with the skin.

Now you may be asking, "how would this work in the 7 menu? Since any value above 18 isn't accessible, or a 4k only arrowskin could crash the game in >4k sets."

The solution is to add (or use the already existing one if there is one, I forget) an arrow skin selector, and make the maximum possible to whatever the length of the list of arrowsizes (and other associated text files) is.

Another argument might be "how do you customize the keys on >18k sets?" The way you could access them is either by using a placeholder texture to attend to >18k sets, or have to switch to the arrowskin in the selector in order to reach the higher key values.

**Comment by Leather128** [comment]
oh yea i put this in already
